typings/ipc.d.ts

```diff
@@ -6,9 +6,12 @@
  * The second is a symlink to the first.
  */
 export interface Options {
+  authed: boolean
   base: string
+  csStaticBase: string
   disableTelemetry: boolean
   disableUpdateCheck: boolean
+  logLevel: number
```

Comment:
This may break some tests in `util.test.ts` 🤔 https://github.com/cdr/code-server/blob/main/test/unit/util.test.ts#L127-L191

Reply:
These properties were actually always there, just not in the types, so we might be OK.